completions/awscli.completion.sh

```diff
@@ -1 +1 @@
-[[ -x "$(command -v aws_completer)" ]] &>/dev/null && complete -C "$(command -v aws_completer)" aws
+[[ -x "$(_omb_util_command_exists aws_completer)" ]] &>/dev/null && complete -C "$(_omb_util_command_exists aws_completer)" aws
```

Comment:
```suggestion
_omb_util_command_exists aws_completer && complete -C "$(_omb_util_command_exists aws_completer)" aws
```

`[[ -x $(command -v CMD) ]] &>/dev/null` needs to be replaced by `_omb_util_command_exists CMD` because `[[ -x $(...) ]]` assumes that the command `...` outputs the path of the binary file and checks the executable bit in the permission of the file. On the other hand, `_omb_util_command_exists` doesn't output anything to `stdout`. I think we actually don't need to check the executable bit because if it doesn't have the executable bit it will never be detected by `command -v` nor `type`.

Could you also replaced the similar cases?

Reply:
Yes I will go through and replace the similar cases for this. Thank you for the guidance!